### PR TITLE
CONN-3331 Update Source Page Title

### PR DESCRIPTION
**Pull request by carlos-castillo-jask**
## Purpose of this pull request

The official name of the source needs to be updated

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

CONN-3331

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/config-based-source.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
 
-With our configuration-based (_config-based_) cloud source, you can collect log data from vendor APIs with a modular configuration. The goal of this source is for Sumo Logic to expand the configuration modules over time giving greater compatibility with vendor APIs, but to acknowledge complex APIs will still require a specific cloud source and not be compatible with this source.
+With our Universal Connector (_config-based_) cloud source, you can collect log data from vendor APIs with a modular configuration. The goal of this source is for Sumo Logic to expand the configuration modules over time giving greater compatibility with vendor APIs, but to acknowledge complex APIs will still require a specific cloud source and not be compatible with this source.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/config-based-source.md:17]: @carlos-castillo-jask Should you delete `(_config-based_)`?

**Review by kimsauce — CHANGES_REQUESTED** [review]
You'll also need to edit lines 46 and 280, where the term "config based" appears.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/config-based-source.md (hunk 1 above)
Comment (carlos-castillo-jask) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/config-based-source.md:17]: I was wondering that too. It seems like we should remove it, so I'll push a change to do so.

**Comment by carlos-castillo-jask** [comment]
> config based

Thanks for catching those. I updated line 46 with the new name. Line 280 actually needs to rename `Config Based` for the `sourceType` as the backend name is the same. We are only changing the UI alias name.

**Comment by carlos-castillo-jask** [comment]
@kimsauce can we get this approved if there are no other issues? Thanks.

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
> can we get this approved if there are no other issues? Thanks.

@carlos-castillo-jask done!

**Pull request merged**